### ISSUE TITLE
Comments om variant - all are global? BUG???

### ISSUE THREAD

**Issue by KickiLagerstedt** [Clinical-Genomics/scout#982]
Analyzing a case and a comment is written on my variant. I open the variant and ca see a comment I have written regarding another case - this is not suppose to be happening!!!

https://scout.scilifelab.se/cust002/F0012595/fb25475084db5a36742af698e167fafb

**Comment by KickiLagerstedt** [comment]
om = on....

**Comment by KickiLagerstedt** [comment]
Another example
https://scout.scilifelab.se/cust002/F0011704/74928d6eaf0cbd72d8a1a27456122911

**Comment by dnil** [comment]
Well found!

**Comment by northwestwitch** [comment]
I'll check. I was recently fixing a big on the local-global variants and I might have introduced another. Thanks for the examples!

**Comment by northwestwitch** [comment]
Ok, I checked this on scout stage, where the code is updated after my bug fix (merged in to master about one month ago) and I couldn't reproduce this error. 
For instance I added a specific comment on variant: 
https://scout-stage.scilifelab.se/cust002/F0011704/74928d6eaf0cbd72d8a1a27456122911

and another on variant: 
https://scout-stage.scilifelab.se/cust002/F0012595/9c5458bf686b26647e28298d584a013d

From each variant I can only see the the comment for that specific variant, and not the other. Global variants on the other hand can be seen by both variants view.

The problem here is that the scout production is not yet updated with the fixed code. We are planning to do the update tomorrow, so this bug shouldn't be found after (hopefully!). But we could check the links after the update and then close the issue, to be on the safe side!
